shared/src/common/enums/exceptions/exception-message.enum.ts

```diff
@@ -0,0 +1,12 @@
+enum ExceptionMessage {
+  UNKNOWN_ERROR = 'Unknown error occurred.',
+  EMAIL_ALREADY_EXISTS = 'Email is already taken.',
+  INCORRECT_EMAIL = 'Incorrect email.',
+  BAD_CREDENTIALS = 'You have entered an invalid username or password.',
+  UNAUTHORIZED_USER = 'Unauthorized user.',
+  INVALID_TOKEN = 'Token is invalid.',
+  PASSWORDS_NOT_MATCH = 'Passwords do not match.',
```

Comment:
we can substitute PASSWORDS_NOT_MATCH and INCORRECT_EMAIL with BAD_CREDENTIALS
[https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html#authentication-responses](https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html#authentication-responses)

Reply:
thanks for the reference to the docs, it was helpful